Dockerfile.redhat

```diff
@@ -408,8 +408,9 @@ WORKDIR /
 SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 # hadolint ignore=DL3003,DL3041,SC2164
 RUN if [ -f /usr/bin/dnf ] ; then export DNF_TOOL=dnf ; else export DNF_TOOL=microdnf ; fi ; \
-        $DNF_TOOL upgrade --setopt=install_weak_deps=0 -y ; \
-        $DNF_TOOL install -y pkg-config && rpm -ivh https://vault.centos.org/centos/8/AppStream/x86_64/os/Packages/tbb-2018.2-9.el8.x86_64.rpm && \
+        echo -e "max_parallel_downloads=8\nretries=50" >> /etc/dnf/dnf.conf ; \
```

Comment:
this doesn't work on ubi-minimal as there is no dnf.conf It make the microdnf unusable